libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.html

```diff
@@ -1,26 +1,77 @@
-<button
-  class="sky-help-inline"
-  type="button"
-  [attr.aria-label]="
-    labelText
-      ? ('skyux_help_inline_aria_label' | skyLibResources: labelText)
-      : ariaLabel
-        ? ariaLabel
-        : ('skyux_help_inline_button_title' | skyLibResources)
-  "
-  [attr.aria-controls]="
-    ariaControls | skyHelpInlineAriaControls: popoverId : helpKey
-  "
-  [attr.aria-expanded]="
-    ariaExpanded | skyHelpInlineAriaExpanded: ariaControls : isPopoverOpened
-  "
-  [attr.aria-haspopup]="helpKey | skyHelpInlineAriaHaspopup"
-  [ngClass]="{
-    'sky-help-inline-hidden': helpKey && !helpSvc && !popoverContent
-  }"
-  [skyPopover]="popoverContent ? popover : undefined"
-  (click)="onClick()"
->
+@if (popoverContent) {
```

Comment:
I separated the buttons into three types: popover triggers, help key triggers, and user-defined (neither popover nor help key). The `@if` checks are required to prevent the `sky-popover` component from being added to the page for cases that don't need the popover.

After reviewing the ARIA pipes, it seemed to me that the logic branches on `popoverContent` and `helpKey`. In some cases, the logic was only applicable to one branch, so I removed the pipes so that the logic is more declarative in the template.